sp/src/game/server/ez2/npc_zassassin.cpp

```diff
@@ -771,24 +748,6 @@ void CNPC_Gonome::AttackSound( void )
 	EmitSound( filter, entindex(), "Gonome.Attack" );	
 }
 
-//=========================================================
-// GrowlSound
-//=========================================================
-void CNPC_Gonome::GrowlSound( void )
-{
-	CPASAttenuationFilter filter( this );
-	EmitSound( filter, entindex(), "Gonome.Growl" );
```

Comment:
Gonome's implementation of GrowlSound does not add a delay. How does this work in practice with the delay?

Reply:
This does cause gonomes to not play this sound if they perform two claw attacks in immediate succession, in accordance with the delay. The sound plays normally after the delay, but I didn't consider this. Do you think that change is significant enough to revert this part?

Reply:
I think I know what you're talking about now. When the gonome swings, it makes this sound twice in a row corresponding to each swing. I didn't realize that was the growl sound. This might just be me but I think not hearing the second sound would be unusual. How did you think it sounded in-game?

Reply:
It did sound a bit strange. I think I'll restore this function along with adding the comment we talked about